diagnosis/getkuberneteslogs.sh

```diff
@@ -134,15 +176,41 @@ else
     || { echo "The identity file $IDENTITYFILE is not a RSA Private Key file."; echo "A RSA private key file starts with '-----BEGIN [RSA|OPENSSH] PRIVATE KEY-----''"; exit 1; }
 fi
 
+if [ -z "$SPN_CLIENT_ID" -a -z "$SPN_CLIENT_SECRET" ] && [ -n "$UPLOAD_LOGS" ]
+then
+    echo ""
+    echo "[ERR] Service Principle details should be provided when upload-storage-account is set to Y"
```

Comment:
Typo `Principle`

Flag name inconsistent: `... provided if logs are stored in a storage account`

Reply:
make the messages below consistent with this one